<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <family>Courier10 BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border: 1px solid black;
	background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLCDNumber" name="timeStarted">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Georgia</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Time Logged In</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLCDNumber" name="currentTime">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Georgia</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Clock Time</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLCDNumber" name="timeSpent">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Georgia</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Time Elapsed</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTextEdit" name="logMessage">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextEdit{
	background-color: white;
	border: 1px solid black;
	border-radius: 15px;
	color: black;
}</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="submitButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    border: 1px solid blue;
	border-radius: 20px;
    padding: 0px;
	color: black;
}

QPushButton:hover {
	background-color: transparent;
    border: 1px solid red;
	border-radius: 20px;
    padding: 0px;
}

QPushButton:pressed {
    background-color: rgba(0, 255, 0, 60);
	border: 2px solid black;
	border-radius: 20px;
	font: 15pt &quot;Georgia&quot;;
    padding: 15px;
}

QPushButton::disabled {
	background-color: rgba(212, 208, 200, 80);
	border: 1px solid grey;
	border-radius: 20px;
	color: grey;
}</string>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QTimeEdit" name="leaveTimeEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="displayFormat">
          <string>HH:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLCDNumber" name="countdown">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Time Remaining for Desired</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLCDNumber" name="timeRemaining">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Georgia</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Time Remaining for Ideal</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="exportToExcelAction">
   <property name="text">
    <string>Export to Excel</string>
   </property>
  </action>
  <action name="fetchDescriptionAction">
   <property name="text">
    <string>Fetch Description</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
